### DISCUSSION TITLE
Режим наблюдателя

### DISCUSSION THREAD

**Discussion by MrTip8** [Sokol-Games-GmbH/PixelCombats.PublicApi#240] [Ideas]
Здарова Костя! Готовлю турик со стримами и комментированием игр... Для хорошего комментирования нужно доработать режим наблюдателей. Вот мой концепт, по возможности сделай пожалуйста до марта.
ТАККЖЕ ПРЕДЛАГАЮ ДОБАВИТЬ ВОЗМОЖНОСТЬ НАБЛЮДЕНИЯ ТОЛЬКО В РЕЖИМ ЗАКЛАДКИ БОМБЫ(В ЭТОМ РЕЖИМЕ БУДЕТ ПРОХОДИТЬ ТУРНИР) И ТОЛЬКО В ЗАКРЫТОМ(ЗАПАРОЛЕНОМ) СЕРВЕРЕ. ТАК ЖЕ ПРЕДЛАГАЮ ДОБАВИТЬ СЧЕТЧИК НАБЛЮДАТЕЛЕЙ(СКОЛЬКО ИХ ИМЕННО).
При нажатии на игрока N(1,2,3,4 или 5) установить камеру от первого лица.
Спасибо и удачи!

![Picsart_24-02-05_15-28-38-213](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/138765207/d27c3de3-b96f-49c1-babb-b9f3608cd75b)
![Picsart_24-02-05_15-27-12-209](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/138765207/66fd306d-2703-4667-bb19-96cf2369993e)
![1666758303738_072817](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/138765207/a5d99f36-a886-4fd6-add7-030338b06a13)

**Comment by kkohno** [comment]
норм идея, однако пока сделать от первого лица не получится - нам нужно систему аваторов отррефакторить сперва... Это будет потом, когда сделаем переключение игры от 3 лица и 1 лица

**Reply by MrTip8** [reply]
Чисто переключения между MapCameraPoints сможешь сделать?
